### DISCUSSION TITLE
is there a csk for visionary T-mini?

### DISCUSSION THREAD

**Discussion by TomerErezSick** [SICKAppSpaceCodingStarterKit/.github#10] [General]
I have seen one on gitlab which i currently work with for an application but not on github.
also the one on gitlab is not very generic. it's built to measure boxes.
is there another version that i missed?

**Comment by golluroSICKAG** [comment]
Hi Tomer,

thx for your question and sorry for the late reply :/
So far I am not aware of any other version.

Do you have any kind of requirement list what features this module should provide?
And how urgent is this topic for you?

Kind regards!
Roy

**Reply by TomerErezSick** [reply]
i made modifications to the currently existing one to fit my application, so this is not urgent.
however my team is planning to work with visionary T mini and visionary s a lot so these modules would be important for us for future usage.
the requirements are plane estimation/ calibration for automatic transformation of the sensor position, and controlling the acquisition filters and camera configs, such as remission filter, intensity, height filters, edge correction etc. 
other than that i think the current csk has enough functionality.
